rfcs/013-release_deployment_tracking/README.md

```diff
@@ -4,396 +4,466 @@
 
 ## Background
 
-We build container images as artifacts for deployment, they are labelled with the git ref at the code point the artifact was packaged.
+We should track what code is deployed where, by whom and for what reason. This will give us a clear picture of the state of our deployments, which is useful for tracking the progress of bugfixes and new features.
 
-Deploying container images to production at present moves all container images forward to the latest built version.
+The build/release/deployment process can be described as follows:
 
-## Problem statement
+![Simple overview](simple_flow.png)
```

Comment:
I know it’s pretty simple, but I really like the `build pushes to release tool` and `deploy reads from release tool`.